package.json

```diff
@@ -8,7 +8,7 @@
   },
   "private": true,
   "scripts": {
-    "build": "redocly bundle -o docs/builds/${npm_config_build_version}.yaml && ./scripts/addBuild.js ${npm_config_build_version} ${npm_config_selected} && ./scripts/applyVersion.js ${npm_config_build_version} docs/builds/${npm_config_build_version}.yaml",
+    "build": "redocly bundle -o docs/builds/${npm_config_build_version}.yaml && ./scripts/addBuild.js ${npm_config_build_version} ${npm_config_selected} && ./scripts/applyVersion.js ${npm_config_build_version} docs/builds/${npm_config_build_version}.yaml && redocly lint",
```

Comment:
Shouldn't `redocly lint` be run before the other commands? That way the build exits early if the spec is invalid.